WcaOnRails/db/seeds/teams.seeds.rb

```diff
@@ -18,3 +18,4 @@
 Team.create(friendly_id: 'vice_chair')
 Team.create(friendly_id: 'wdpc', email: "dataprotection@worldcubeassociation.org")
 Team.create(friendly_id: 'wac', email: "advisory@worldcubeassociation.org")
+Team.create(friendly_id: 'probation', hidden: true)
```

Comment:
You are missing a blank line at the end of this file.

Reply:
Fixed.